src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -14,6 +14,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>RecurringDonations2MigrationState__c</fullName>
```

Comment:
@sam-knox , can you please review the field containing dry run and data migration results?

Reply:
@RadSFDeveloper - Would this label only be visible in the settings object, or also on the RD2 enablement page in NPSP Settings?

Reply:
Only on the RD settings.

Reply:
@RadSFDeveloper - Cool, then I think what you have here works fine.